requirements/dev-requirements.txt

```diff
@@ -1,10 +1,5 @@
 # Dev dependencies
 -r tests-requirements.txt
-attrs>=21.1
-GitPython>=2.0
 setup-cfg-fmt
-yamllint
-setuptools>=62.0.0
 twine
 wheel
-backports.cached_property
```

Comment:
Without this dependency, the linter will complain about it with Python 3.7

Reply:
Don't we have it in `setup.cfg` already though?

Reply:
oh you are right